collect_app/src/main/res/values/strings.xml

```diff
@@ -572,5 +572,5 @@
     <string name="moving_backwards_enabled_message">You may wish to review the following settings:\n\n\u2022 Edit Saved Form\n\u2022 Save Form\n\u2022 Go To Prompt\n\u2022 Constraint processing</string>
     <string name="yes">Yes</string>
     <string name="no">No</string>
-    <string name="newer_version_of_form_warning">This is an updated version of form you already have!</string>
+    <string name="newer_version_of_form_info">This is an updated version of form you already have!</string>
```

Comment:
Should be "of a form" rather than "of form." (I apologize about English having articles, I know Slavic languages don't have them! https://en.wikipedia.org/wiki/Article_(grammar)#Variation_among_languages)